tasks/species_typing/task_ngmaster.wdl

```diff
@@ -7,24 +7,58 @@ task ngmaster {
   input {
     File assembly
     String samplename
-    String docker = "quay.io/biocontainers/ngmaster:0.5.8--pyhdfd78af_1"
-    Int? cpu = 2
+    String docker = "quay.io/staphb/ngmaster:1.0.0"
```

Comment:
original PHBG has a [docker container](https://github.com/theiagen/public_health_bacterial_genomics/blob/main/tasks/species_typing/task_ngmaster.wdl#L10) but we can go with the one hosted on quay.io